ivadomed/utils.py

```diff
@@ -313,6 +313,41 @@ def _version_string():
 __version__ = _version_string()
 
 
+def get_command(args):
+    try:
+        if args.train:
+            return "train"
+        elif args.test:
+            return "test"
+        elif args.segment:
+            return "segment"
+        else:
+            return "train" # default to training
+    except AttributeError:
+        logger.warning("Issue parsing argument for command: ( --train | --test | --segment ). Will default to train command.")
+        return "train"
+
+
+def get_log_directory(args):
+    try:
```

Comment:
See above, I think you might not actually need `get_log_directory` and `get_bids_path`